src/Microsoft.Azure.WebJobs.Extension.Dapr/DaprExtensionConfigProvider.cs

```diff
@@ -139,106 +145,104 @@ static DaprBindingMessage CreateBindingMessage(object paramValues)
 
         static DaprBindingMessage CreateBindingMessage(byte[] paramValues)
         {
-            return CreateBindingMessage(JObject.Parse(System.Text.Encoding.UTF8.GetString(paramValues)));
+            return CreateBindingMessage(BytesToJsonElement(paramValues));
         }
 
-        static DaprBindingMessage CreateBindingMessage(JObject json)
+        static DaprBindingMessage CreateBindingMessage(JsonElement jsonElement)
         {
-            if (!TryGetValue(json, "data", out object? data))
+            if (!jsonElement.TryGetProperty("data", out JsonElement data))
```

Comment:
The output from the jsonElement is not nullable, previously we were using a custom method instead of the built-in [JsonElement's TryGetProperty](https://learn.microsoft.com/en-us/dotnet/api/system.text.json.jsonelement.trygetproperty?view=net-7.0#system-text-json-jsonelement-trygetproperty(system-string-system-text-json-jsonelement@)).


<img width="1114" alt="image" src="https://user-images.githubusercontent.com/22132944/230290945-14c11486-434c-4800-9100-244c064a140a.png">

Or are you recommending something else?